src/librbd/AioObjectRequest.cc

```diff
@@ -410,27 +451,10 @@ namespace librbd {
           		       << m_object_off << "~" << m_object_len << dendl;
         m_state = LIBRBD_AIO_WRITE_PRE;
 
-        uint8_t new_state;
```

Comment:
I am most likely missing something, but what is the point of the new state?  This ```::send_pre``` doesn't appear to do any async (librados) action anymore